CHANGELOG.md

```diff
@@ -33,7 +35,6 @@ Line wrap the file at 100 chars.                                              Th
 #### Windows
 - Fix window flickering by disabling window animations.
 
-
```

Comment:
Please don't remove this line. We keep two empty new lines between versions.

Reply:
Fixed.